lang/LangPrimSource/PyrFilePrim.cpp

```diff
@@ -1326,6 +1326,74 @@ int prPipeOpen(struct VMGlobals *g, int numArgsPushed)
 	return errNone;
 }
 
+int prPipeOpenArgv(struct VMGlobals *g, int numArgsPushed)
+{
+	PyrSlot *a, *b, *c;
+	char mode[12];
+
+	a = g->sp - 2;
+	b = g->sp - 1; //argv
+	c = g->sp;     //mode
```

Comment:
probably better to just give these meaningful names than document their meaning in comments. `callerSlot`, `argvSlot`, `modeSlot` are good choices

Reply:
In other primitives that I looked at these variables are always called a,b,c,etc... Should we change that convention here ?

Reply:
Do you think it's a useful convention? I really dont.

Reply:
Not really...

Reply:
Changed to callerSlot, argvSlot, modeSlot